compiler/resolution/interfaceResolution.cpp

```diff
@@ -833,6 +844,77 @@ static BlockStmt* createHolderBlock(FnSymbol* wrapFn, ImplementsStmt* istm) {
   return holder;
 }
 
+static bool foundTargetFunctionValid(FnSymbol* tg, bool generatedOnly) {
+  (void) generatedOnly;
```

Comment:
I'd like to see a comment why this line `(void) generatedOnly;` is needed. Ideally, it is short enough to fit on the same line.

In the function name, I prefer `IsValid` instead of simply `Valid`. Better yet, simplify to `targetFunctionIsValid`. Assuming that makes sense. Otherwise I would like to see a comment explaining what the function is checking.